host/src/containers/GameSessionContainer.tsx

```diff
@@ -138,22 +166,32 @@ const GameSessionContainer = () => {
         }
       },
     );
-
+      
     // set up subscription for teams answering
     let createTeamAnswerSubscription: any | null = null;
     createTeamAnswerSubscription = apiClient.subscribeCreateTeamAnswer(
       gameSessionId,
       (teamAnswerResponse) => {
-        setTeamsArray((prevState) => {
-          let newState = JSON.parse(JSON.stringify(prevState));
-          newState.forEach((team) => {
-            team.teamMembers &&
-              team.teamMembers.forEach((teamMember) => {
-                if (teamMember.id === teamAnswerResponse.teamMemberAnswersId)
-                  teamMember.answers.push(teamAnswerResponse);
-              });
+        let choices = '';
+        apiClient.getGameSession(gameSessionId).then((response) => {
```

Comment:
this is where we listen for the create team answer subscription. `buildShortAnswerResponses` is the function that performs the equality checks